src/main/java/Storage.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private List<Task> myTasks = new ArrayList<>();
+
+    public void storeTasks(Task task) {
+        myTasks.add(task);
+    }
+
+    public void displayTasks() {
+        Printer.printTasks(myTasks);
+    }
+
+    public Task getTask(int index) {
+        try {
+            return myTasks.get(index - 1);
+        }
+        catch(Exception e) {
+            return null;
```

Comment:
Maybe you can use Printer to give some error messages if any exception has been caught.